### ISSUE TITLE
not changed size

### ISSUE THREAD

**Issue by LetSwiftDev** [Minitour/AZDialogViewController#4]
hello

how can I change window size ? when iPad landscape looking very big window. 

Thank you.

**Comment by Minitour** [comment]
@LetSwiftDev Hi sorry for the late response. Here is a quick-fix solution:

Simply modify these computed variables and you should get a normal size dialog on iPads.
I will include this fix in the next release.
```swift
    // Helper to get the real device width
    fileprivate var deviceWidth: CGFloat {
        let realValue = view.bounds.width < view.bounds.height ? view.bounds.width : view.bounds.height
        return realValue > 414 ? realValue / 2 : realValue
    }
    
    // Helper to get the real device height
    fileprivate var deviceHeight: CGFloat {
        let realValue = view.bounds.width < view.bounds.height ? view.bounds.height : view.bounds.width
        return realValue > 736 ? realValue / 2 : realValue
    }
```

**Comment by Minitour** [comment]
Fixed with 1.1.5
